air-quality-ui/src/components/summary-grid/cell/cell-rules/CellRules.ts

```diff
@@ -0,0 +1,100 @@
+import colourCell, { Params } from './ColourCell'
+
+export default function cellRules(showAllColoured: boolean) {
+  return {
+    pm2_5: {
+      'cell-very-good': (params: Params) =>
+        colourCell(showAllColoured, params, 0, 10),
+      'cell-good': (params: Params) =>
+        colourCell(showAllColoured, params, 10, 20),
```

Comment:
I have a suspicion we have somewhere in the UI that can calculate the AQI level for a pollutant value somewhere else. If not maybe we should add one as it's a fairly useful thing to have, and then we could rework this to use it?
It feels like we're missing a trick if we're embedding an AQI converter into a specific CSS class selector when we could make a nice reusable function to do it instead.

Happy if you'd rather do that as part of the future refactoring or in a separate PR (this one is pretty big already)